values.yaml

```diff
@@ -50,6 +50,13 @@ global:
   # clients within Kubernetes. Additionally requires Consul v1.4+ and
   # consul-k8s v0.8.0+.
   bootstrapACLs: false
+  
+  # updateStrategy used to customize number of pods that can be upgraded simultaniously
+  # starting from version 1.5.1 and using host path to store data between restarts, we can rollout 
+  # consul much faster with 10  maxUnavailable pods for example.
```

Comment:
Use this instead:
```yaml
# updateStrategy for the DaemonSet.
# See https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#daemonset-update-strategy.
# Must be set as a raw YAML string.
updateStrategy: null
```

Reply:
I have problem with YAML raw string. Indentation is wrong.
Best i can do is
```
 {{- if .Values.client.updateStrategy }}
  updateStrategy: {{ .Values.client.updateStrategy | indent 4}}
  {{- end }}
```
and than in values.yaml
```
client:
  updateStrategy: |
    #updateStrategy from values.yaml
    rollingUpdate:
      maxUnavailable: 11
    type: RollingUpdate
```